lib/Dialect/Linalg/Transforms/Tiling.cpp

```diff
@@ -215,9 +215,28 @@ makeTiledViews(OpBuilder &b, Location loc, LinalgOp linalgOp,
   return res;
 }
 
+bool isPermutationValid(const ArrayRef<uint64_t>& permutation) {
```

Comment:
ArrayRef should be passed by value, not by reference.

Reply:
My bad! Done.